app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -29,4 +29,20 @@ document.addEventListener('turbolinks:load', function() {
       }
     });
   }
+
+  $(function() {
```

Comment:
What's the reasoning behind this wrapping of the below functions?

Reply:
Good catch. It left over when trying out things. Removed it